ote/src/cljs/ote/views/place_search.cljs

```diff
@@ -186,8 +194,30 @@
                          :on-update-input #(e! (ps/->SetSecondaryPlaceName %))
                          :search-text (or (:secondary-name place-search) "")
                          :on-new-request #(e! (ps/->AddPlace (aget % "id") false))}]]
+
      [:div {:style {:width "70%"}}
-       [places-map e! (:results place-search)]]]))
+       [places-map e! (:results place-search) (:show? place-search)]
+
+      [:span
```

Comment:
I prefer divs

Reply:
I'll leave this span which Toni recommended using here.